ClassMoodApp/static/tsc/controllers/CreatePollingQuestionController.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference path="../app.ts" />
+declare var angular: ng.IAngularStatic;
+module ClassMoodApp {
+    "use strict";
+    export class CreatePollingQuestionController {
+        static $inject = ["$scope", "$http", "$window", "$routeParams"];
+        public questions:Array<CreatePollingQuestionModel>;
+        public lecture_id: number;
+        public isStudent: boolean;
+        private userId: number;
+        private classId: number;
+        constructor(private $scope: ng.IScope,
+                    private $http: ng.IHttpService,
+                    private $window: ng.IWindowService,
+                    private $routeParams: ng.route.IRouteParamsService) {
+                        this.$http = $http;
+                        this.$scope = $scope;
+                        this.$window = $window;
+                        let curLink = $window.location.pathname.split("/");
+                        this.classId = parseInt(curLink[curLink.length-3]);
+                        this.lecture_id = parseInt(curLink[curLink.length-2]);
+                    }
+
+        public submitPollingQuestion(): void {
+		let text = this.$scope.question_text;
+		let a = this.$scope.q.a_text;
+		let b = this.$scope.q.b_text;
+		let c = this.$scope.q.c_text;
+		let d = this.$scope.q.d_text;
+		let answer = this.$scope.q.ans;
+		if (!this.isStudent) {
+			if (text && a && b) {
+				this.$http.get(`/create_polling_question/${this.lecture_id}/${text}/${a}/${b}/${c}/${d}/${answer}`).then(
+					(response: any) => {
+						if (response.data) {
```

Comment:
Preferably use === for strict equality comparison to bools. Javascript is a bit finicky with booleans.